presto-native-execution/presto_cpp/main/TaskManager.cpp

```diff
@@ -120,6 +120,14 @@ void TaskManager::acknowledgeResults(
 
 namespace {
 
+std::unique_ptr<Result> handleTimeOut(long token) {
```

Comment:
Thank you for extracting a helper function. The name suggests that the method does something to respond to a timeout, but it actually just generated a timeout response (without, for example, sending it out). Consider renaming for clarity: createTimeoutResult or getTimeoutResult or makeTimeoutResult.